src/applications/caregivers/definitions/caregiverUI.js

```diff
@@ -0,0 +1,196 @@
+import * as address from 'platform/forms-system/src/js/definitions/address';
```

Comment:
I broke out all UI elements so I could just look in one consolidated file to edit content, and see UI elements. IMO makes the config file more readable.

Reply:
For me, I moved all the `schema` and `uiSchema` sections for each page into separate files. The `config/form.js` ends up looking like this:

```js
veteranInfoOne: {
  path: 'service-member',
  title: 'Veteran Information',
  uiSchema: vetFields.uiSchema,
  schema: vetFields.schema,
},
```

This makes it so I rarely have to modify the form config.